test/e2e/storage/csi_objects.go

```diff
@@ -427,3 +437,50 @@ func deployGCEPDCSIDriver(
 	framework.ExpectNoError(err, "Failed to create DaemonSet: %v", nodeds.Name)
 
 }
+
+func createCSICRDs(c apiextensionsclient.Interface) {
+	By("Creating CSI CRDs")
+	crds := []apiextensionsv1beta1.CustomResourceDefinition{
```

Comment:
Please don't check CRDs in like this. (See my comments on https://github.com/kubernetes/kubernetes/pull/68406)

Please also don't check in multiple copies of the same CRD.